flood_forecast/transformer_xl/dummy_torch.py

```diff
@@ -13,5 +13,5 @@ def __init__(self, forecast_length:int):
 
     def forward(self, x:torch.Tensor, mask=None):
         batch_sz = x.size(0)
-        result = torch.ones(batch_sz, self.out_len, requires_grad=True)
+        result = torch.ones(batch_sz, self.out_len, requires_grad=True, device=x.device)
```

Comment:
Was this for CUDA compatibility?

Reply:
yeap-- when I was running it locally the tensors are not in the same device as the model